.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-07-09 23:34:13 +0200 using RuboCop version 0.41.2.
+# on 2016-10-13 17:03:27 +0530 using RuboCop version 0.43.0.
```

Comment:
For what it's worth, I did not have to make any changes to the rubocop todo file. Is this change required?